### DISCUSSION TITLE
Words/Replies consistently cut off... and "Settings" on right?

### DISCUSSION THREAD

**Discussion by SweetLioness** [LAION-AI/Open-Assistant#3335] [Q&A]
Hi. Open Assistant always cuts messages off after a few paragraphs. Random numbers of words. Once I asked it for word count on two cut off messages: One was 394 words and another about 500 words.

What is happening?

Also, how can I find out what the selections on the right do? (Model, Preset, Plugins...)

**Comment by someone13574** [comment]
Its most likely hitting the token limit of 1024. A token isn't always a full word which would explain the different word counts you were getting, one message probably had longer words in it meaning it hit the token limit faster. At the moment  there isn't a way to prevent it really other than trying to get the model to continue its previous message. In the future a feature to continue generating a message may be added, but currently no one is working on it. #2945

**Reply by SweetLioness** [reply]
Thanks, that makes sense. Having them continue had/has always been about 70% effective.

Any idea how Model/Preset/Plugins affect the experience?
